src/vsDebugServer.ts

```diff
@@ -126,15 +126,34 @@ class VsDebugServer implements ISessionLauncher<VSDebugSession> {
   }
 }
 
-const debugServerPort = process.argv.length >= 3 ? +process.argv[2] : undefined;
+let debugServerPort: number | undefined = undefined;
+let debugServerHost: string | undefined = undefined;
+if (process.argv.length >= 3) {
+  // Interpret the argument as either a port number, or 'address:port'.
+  const address = process.argv[2];
+  const colonIndex = address.lastIndexOf(':');
```

Comment:
I suggest that we don't do this, since this won't work if the user wants to listen on an ipv6 interface. (We could do better parsing, but I think having separate arguments is fine, especially since this CLI is mostly consumed by machines and not humans)

Reply:
I've seen a similar support of `:` parsing in another adapter entry point (`debugServerMain.ts` I think?), so I reused the same code. However, as it is rather easy to change now and will prevent future breaking of the command line, changing it now seems like the right thing to do 👍